send/pithia_portal

```diff
@@ -1,4 +1,42 @@
-#!/bin/bash
-export SERVICE_NAME="pithia_portal"
+#!/usr/bin/env python3
+import os
+import sys
+import subprocess
+import send_lib
+import requests
+import generic_sender
 
-python3 generic_send.py "$1" "$2" "$3"
+SERVICE_NAME = "pithia_portal"
+os.environ["SERVICE_NAME"] = SERVICE_NAME
+
+send_lib.check_input_fields(sys.argv, True)
+
+facility = sys.argv[1]
+destination = sys.argv[2]
+destination_type = sys.argv[3]
+
+send_lib.check_destination_type_allowed(destination_type, send_lib.DESTINATION_TYPE_URL)
+send_lib.check_destination_format(destination, destination_type)
```

Comment:
Check on the destination format is part of the `generic_sender.send()` implementation. Check on allowed destination type is a good catch, since it's specific to this service.